### PR TITLE
[master] Add timeout for poweroff and restart VM

### PR DESCRIPTION
**Pull request by SeanHQF**
So that it won't hang forever.

Signed-off-by: QingFeng Hao <haoqf@linux.vnet.ibm.com>

Conflicts:
	smtLayer/vmUtils.py

### CODE REVIEW

**Comment by SeanHQF** [comment]
@jichenjc @bjhuangr pls help review. thx
Currently the test passes the happy path but I can't simulate the timeout exception even I changed the timeout=1 in the code when doing shutdown or restart. Will continue testing.

**Comment by SeanHQF** [comment]
Pls help review thanks

File: smtLayer/powerVM.py [hunk 1]
```diff
@@ -538,7 +538,7 @@ def reboot(rh):
     rh.printSysLog("Enter powerVM.reboot, userid: " + rh.userid)
 
     strCmd = "shutdown -r now"
-    results = execCmdThruIUCV(rh, rh.userid, strCmd)
+    results = execCmdThruIUCV(rh, rh.userid, strCmd, timeout = 60)
```
Comment (jichenjc) [inline smtLayer/powerVM.py:541]: is timeout 60 good enough？

Comment (SeanHQF) [reply smtLayer/powerVM.py:541]: I think so because normally even 1 second can make it work and that's why I can't reproduce the problem.

Comment (SeanHQF) [reply smtLayer/powerVM.py:541]: @jackydalong pls help review, thanks

**Comment by SeanHQF** [comment]
@haoxy000

**Review by jackydalong — APPROVED** [review]
lgtm

File: smtLayer/powerVM.py (hunk 1 above)
Comment (bjhuangr) [reply smtLayer/powerVM.py:541]: `timeout = 60` -> `timeout=60`

**Comment by SeanHQF** [comment]
Is it ok to merge? @jichenjc @bjhuangr thanks

**Pull request merged**